cognite_toolkit/_cdf_tk/_parameters/type_hint.py

```diff
@@ -86,7 +86,10 @@ def is_any(self) -> bool:
 
     @property
     def is_user_defined_class(self) -> bool:
-        return any(inspect.isclass(arg) and arg.__module__ not in {"typing", "builtins"} for arg in self.args)
+        return any(
+            inspect.isclass(arg) and arg.__module__ not in {"typing", "builtins", "collections.abc"}
```

Comment:
The new Kafka source in the cognite-sdk uses `Sequence` from `collections.abc` and not `typing` which caused this to fail on 3.10